BEIMA.Client/src/pages/DeviceTypes/AddDeviceTypePage.js

```diff
@@ -82,7 +83,13 @@ const AddDeviceTypePage = () => {
     let fieldsJSON = {"fields" : customDeviceFields};
     fullTypeJSON = Object.assign(attributeValues, fieldsJSON);
     setCustomDeviceFields([]);
-    await AddDeviceType(fullTypeJSON);
+    let addResult = await AddDeviceType(fullTypeJSON);
```

Comment:
I think that I had mentioned this in the devices PR, but I think that the name of this function ("createJSON") is a little misleading since it is both creating the JSON and calling the endpoint. I think we should either split this logic into two properly named functions or rename this function. However we did it in devices I guess.

Reply:
It's split in AddDevice like:
  function saveDeviceToDb(addButtonEvent) {
    const formFields = addButtonEvent.target.form.elements;
    const dbJson = createJSON(formFields);
    ...
  }

around line 167

Reply:
I've split this function up. Copied how we were doing it in the add devices page